api/api.go

```diff
@@ -97,5 +99,29 @@ func (a *Api) Run() {
 		a.EventHandler,
 	})).Methods("GET")
 
+	if a.DebugUI {
+		log.Info("ui: local debug mode (from /ui/dist)")
+		routes.PathPrefix("/dist").Handler(a.MWHandler.Handle([]rye.Handler{
```

Comment:
The `/dist` path is for static resources such as CSS, font files, images, etc. Webpack compiles them into the dist folder. However, this path needs to be separate from the `index.html` file that gets served by the `/ui` path below. The `/ui` path serves ONLY ONE FILE: `index.html` and the `bundle.js` does the rest -> it's the compiled JavaScript single-page application.